subsys/mpsl/clock_ctrl/Kconfig

```diff
@@ -48,4 +48,11 @@ config MPSL_EXT_CLK_CTRL_LFCLK_REQ_TIMEOUT_ALLOW
 	  for a clock with better accuracy than the one requested by the MPSL.
 	  It is user responsibility to make sure these requirements are fulfilled.
 
+config MPSL_CLOCK_CTRL_CALLBACK
+	bool "Enable callback notification API"
+	default n
```

Comment:
There's no need to specify `default n`, as kconfig booleans are not selected by default anyway